libs/vertexai/langchain_google_vertexai/vectorstores/vectorstores.py

```diff
@@ -192,6 +192,10 @@ def add_texts(
                 f"{len(metadatas)} != {len(texts)}"
             )
 
+        # Add generated IDs to metadata
```

Comment:
I'm not sure modifying things in-place is a good idea. Can we just pass doc_ids from outside maybe?